External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -26,6 +26,8 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %   bOverwrite     Vector with 3 Booleans for overwriting the following pre-existing destination files:
 %                  1) NIfTI file, 2) JSON file, 3) mat-orientation motion file (for 4D NIfTIs) (OPTIONAL, DEFAULT = [1 1 1]; 
 %                  1 -> [1 1 1] and 0 -> [0 0 0] and creates a warning; [1 0] - error
+%   changeMat0     New orientation matrix mat0 4x4. Note that mat0 should always stay as it was originally from the scanner, 
+%                  but we can exceptionally change it if mat0 was completely wrong (OPTIONAL, DEFAULT same as previous)

```

Comment:
instead of "exceptionally change it" (which is not grammatically correct I believe :) I would write:

"but in rare cases we change it if the orientation matrix was not correctly converted from the DICOM header."

Reply:
Fixing